crates/continuations/src/lib.rs

```diff
@@ -87,11 +88,11 @@ pub struct Payloads {
 }
 
 impl Payloads {
-    pub fn new(capacity: usize) -> Self {
+    pub fn new(capacity: u32) -> Self {
         let data = if capacity == 0 {
```

Comment:
I think we can simplify this code. I think a `vec` of size `0` is guaranteed to occupy zero memory.